api/Worker.json

```diff
@@ -116,7 +116,13 @@
             "nodejs": {
               "version_added": "10.5.0",
               "partial_implementation": true,
-              "notes": "Takes an entirely different options object."
+              "notes": "Takes an entirely different options object.",
+              "flags": [
+                {
+                  "type": "runtime_flag",
+                  "name": "--experimental-worker"
+                }
+              ]
             },
```

Comment:
If we're carrying the flag data down, we should separate these into multiple statements, like so:

```json
"nodejs": [
  {
    "version_added": "11.7.0",
    "partial_implementation": true,
    "notes": "Takes an entirely different options object."
  },
  {
    "version_added": "10.5.0",
    "partial_implementation": true,
    "notes": "Takes an entirely different options object.",
    "flags": [
      {
        "type": "runtime_flag",
        "name": "--experimental-worker"
      }
    ]
  }
]
```

(This applies to all of the NodeJS changes in this file.)